File: exercises/py101/easy1_small_problems/tip_calculator.py


bill = float(input("What is the bill? "))
tip_percentage = float(input("What is the tip percentage? "))

tip = bill * (tip_percentage / 100)
total = bill + tip 

print(f"The tip is ${tip:.2f}")
print(f"The total is ${total:.2f}")

File: exercises/js210/easy1/tip_calculator.js

const readlineSync = require('readline-sync');

let bill = parseFloat(readlineSync.question('What is the bill? '));
let tipPercentage = parseFloat(readlineSync.question('What is the tip percentage '));

let tip = bill * tipPercentage / 100;
let total = bill + tip;

console.log(`The tip is $${tip.toFixed(2)}`);
console.log(`The total is $${total.toFixed(2)}`);